packages/react/src/themes/teams/components/Input/inputVariables.ts

```diff
@@ -1,41 +1,17 @@
+import { InputVariables } from '../../../base/components/Input/inputVariables'
 import { pxToRem } from '../../../../lib'
 
-export interface InputVariables {
-  backgroundColor: string
-  borderRadius: string
-  borderWidth: string
-  fontColor: string
-  fontSize: string
-  iconColor: string
-  iconPosition: string
-  iconRight: string
-  iconLeft: string
-  inputPaddingWithIconAtStart: string
-  inputPaddingWithIconAtEnd: string
-  inputPadding: string
-  inputFocusBorderBottomColor: string
-  inputFocusBorderRadius: string
-  placeholderColor: string
-}
-
-export default (siteVars): InputVariables => ({
-  backgroundColor: siteVars.gray10,
+export default (siteVars): Partial<InputVariables> => ({
+  borderColor: 'transparent',
   borderRadius: `${pxToRem(3)} ${pxToRem(3)} ${pxToRem(2)} ${pxToRem(2)}`,
   borderWidth: `0 0 ${pxToRem(2)} 0`,
+  backgroundColor: siteVars.gray10,
 
   fontColor: siteVars.gray02,
   fontSize: siteVars.fontSizes.medium,
 
-  iconPosition: 'absolute',
-  iconRight: pxToRem(10),
   iconColor: siteVars.bodyColor,
-  iconLeft: pxToRem(6),
-  inputPaddingWithIconAtStart: `${pxToRem(7)} ${pxToRem(12)} ${pxToRem(7)} ${pxToRem(24)}`,
-  inputPaddingWithIconAtEnd: `${pxToRem(7)} ${pxToRem(24)} ${pxToRem(7)} ${pxToRem(12)}`,
-
-  inputPadding: `${pxToRem(7)} ${pxToRem(12)}`,
-  inputFocusBorderBottomColor: siteVars.colors.primary[500],
+  inputFocusBorderColor: `transparent transparent ${siteVars.colors.primary[500]} transparent`,
```

Comment:
I hate that we are using `input` as part of the variables name, but the `input` is a shorthand of the `Input` component, so I don't see any better way... :(

Reply:
I don't have better ideas there, too :(